networking/network_observability/network-observability-overview.adoc

```diff
@@ -36,6 +40,8 @@ The Network Observability Operator provides the Flow Collector API custom resour
 
 The {product-title} console offers the *Overview* tab which displays the overall aggregated metrics of the network traffic flow on the cluster. The information can be displayed by node, namespace, owner, pod, and service. Filters and display options can further refine the metrics.
 
+Additionally, the *Observe* tab offers the *Dashboards* menu. In the *Netobserv/Health* dashboard, you can access metrics about health and resource usage of the Network Observability Operator. In the 
```

Comment:
I think something is missing at the end here.

Reply:
Looks like I accidentally committed the changes in this file but they were meant for #60060. Sorry, trying to switch between too many PRs at the same time.